requirements-test.txt

```diff
@@ -3,6 +3,7 @@ coverage<5
 codecov
 pytest-mypy
 pytest-cov
+git+https://github.com/sbrugman/pluggy.git@patch-1
```

Comment:
Is this being PRed into pluggy? Can we wait until it's in the source repository?